doc_src/cmds/if.rst

```diff
@@ -6,9 +6,9 @@ if - conditionally execute a command
 Synopsis
 --------
 
-| **if** *CONDITION*; *COMMANDS_TRUE*...;
-| [**else** **if** *CONDITION2*; *COMMANDS_TRUE2*...;]
-| [**else**; *COMMANDS_FALSE*...;]
+| **if** *CONDITION*; *COMMANDS_TRUE* ...;
```

Comment:
I would prefer if this was formatted so it shows `if` used as a block.

I'm not entirely sure how that would work, maybe

```rst
| **if** *CONDITION*
|            *COMMANDS_TRUE* ...
| [**else** **if** *CONDITION2*
|            *COMMANDS_TRUE2* ...]
| [**else**
|            *COMMANDS_FALSE* ...]
| **end**
```

?

Reply:
Yeah, this was also suggested in https://github.com/fish-shell/fish-shell/pull/8626#discussion_r780795608, so we all agree that the block version looks better.
I left this for a follow-up though.

Your example works but uses 2-space indentation. Making the columns align is actually tricky to get right (see e4665fe0c (docs synopses: fix alignment of continuation lines, 2022-01-09)).
This will work (the weird placement of the backslash is critical):

```rst
| **if** *CONDITION*
       \     *COMMANDS_TRUE* ...
| [**else** **if** *CONDITION2*
       \     *COMMANDS_TRUE2* ...]
| [**else**
       \     *COMMANDS_FALSE* ...]
| **end**
```

If this is too complicated we could go back to literal code blocks (with something like #8625).